src/components/Auth/RequireAuth.js

```diff
@@ -44,13 +44,13 @@ const requireAuth = (WrappedComponent, requiredState) => {
     }
   }
 
-  const getPermissions = (permissions) => {
-    return permissions;
+  const getUser = (user) => {
```

Comment:
That's how they did it in the Redux documentation. For my purposes it's probably not necessary, but if you want to transform the store before mapping it to component props you'd do something like this.